public/js/template.js

```diff
@@ -0,0 +1,38 @@
+jQuery(document).ready(function($) {
+
+	var my_nav = $('.navbar-sticky'); 
+	// grab the initial top offset of the navigation 
+	var sticky_navigation_offset_top = my_nav.offset().top;

```

Comment:
Codacy found an issue: [Identifier 'sticky_navigation_offset_top' is not in camel case.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6084628)